app/client/components/cancel/stages/areYouSure.tsx

```diff
@@ -0,0 +1,9 @@
+import { Link } from "@reach/router";
+import * as React from "react";
```

Comment:
We should probably standardise on `import React from "react"`

Reply:
done